sphinx/config.py

```diff
@@ -169,7 +169,9 @@ class Config(object):
         epub_theme = ('epub', 'html'),
         epub_theme_options = ({}, 'html'),
         epub_title = (lambda self: self.html_title, 'html'),
+        epub_description = ('', 'epub3', [str]),
```

Comment:
I think config variables for `epub3` builder should be start with `epub3_` suffix.
Or not, it might confuse users.